docs/versioning.md

```diff
@@ -64,7 +64,7 @@ more structured way in [feature-gates.md](feature-gates.md).
 
 ## Release cycle
 
-New Antrea minor releases are currently shipped every 6 to 8 weeks. This fast
+New Antrea minor releases are currently shipped every 12 to 14 weeks. This fast
```

Comment:
I would remove the "fast" qualifier here
Which is fine, the project is more mature now so we don't need / want to release as often

Reply:
Removed.